src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -259,7 +245,7 @@ module V = struct
   include Hashable.Make (T)
 end
 
-type 'f t =
+type ('f, 'rust_gates) t =
```

Comment:
I have a comment at the very bottom of this commit to explain why there's a new parameter